.changeset/strong-plants-shake.md

```diff
@@ -0,0 +1,10 @@
+---
+"@pnpm/config": minor
+"@pnpm/plugin-commands-installation": patch
+---
+
+A new setting is returned by `@pnpm/config`: `npmGlobalBinDir`.
```

Comment:
Does this mean that everyone for whom this is currently failing will have to manually specify the global bin dir?